src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -2511,6 +2513,35 @@ public void onSignLoads(ChunkLoadEvent event) {
       }
     }
   }
+  
+  @EventHandler(priority = EventPriority.HIGHEST)
+  public void adminAccessBlockedChest(PlayerInteractEvent e) {
+	  if (!e.getPlayer().isOp()) {
```

Comment:
also add a permission check here if you could as well.  Otherwise looks perfect.